tests/TestParser.elm

```diff
@@ -573,12 +572,12 @@ expectRawAst subject expected =
 
 expectValue : String -> Ast.Value -> Expect.Expectation
 expectValue subject expected =
-    case ( Parser.fromString subject, expected ) of
-        ( Ok (Ast.Float_ got), Ast.Float_ want ) ->
-            expectCloseTo got want
+    case (Parser.fromString subject, expected) of
+        (Ok got, _) ->
+            Expect.equal got expected
 
-        ( got, _ ) ->
-            Expect.equal got (Ok expected)
+        (Err err, _) ->
+            Expect.fail err
```

Comment:
This is incorrect. Comparing floating point values for equality should be done more carefully. Also, such a change should be done in another PR since it is unrelated to colon keys. Once again, I think I see where you're going and, in principle, I would accept this function being cleaned up a bit but please do this in a separate RP.